frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -264,15 +266,15 @@ function ReaderCoptListener:getAltStatusBarMenu()
             },
             {
                 text_func = function()
-                    local status = _("off")
+                    local status = _("Off")
                     if self.battery == 1 then
                         if self.battery_percent == 1 then
-                            status = _("percentage")
+                            status = _("Percentage")
                         else
-                            status = _("icon")
+                            status = _("Icon")
                         end
                     end
-                    return T(_("Battery status (%1)"), status)
+                    return T(_("Battery status: %1"), status)
```

Comment:
Dunno if text following a colon should be capitalized. In French, they don't - feels like they shouldn't in English, but not sure. @Frenzie @NiLuJe ?

Reply:
Thank's, that is  a good point.
So my question is. How can we tell our translators to or not to capitalize that? Or do they now it out on the head?
Can we give them a hint, that these strings come after a colon?

For this PR only enlish counts: English Professors?

Reply:
Because these should really be capitalized in German ? And if yes, because of the colon ? or because german nouns are capitalized (if I remember right) and these (Icons, Percentage) are nouns. What it it is an adjective (thick, thin) ? How does `Size: thick` look in German ?

Reply:
Uuuuuuupppppppppsssss: You really got me ;( ;(
Of course `Size: 20, bold` has to be `Größe: 20, fett`.

It's time to retire for me :)

Reply:
Normally you wouldn't capitalize following a colon.